src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -2578,670 +2620,1067 @@ void gfx_set_framebuffer(int fb, float noise_scale);
 void gfx_reset_framebuffer();
 void gfx_copy_framebuffer(int fb_dst_id, int fb_src_id, bool copyOnce, bool* hasCopiedPtr);
 
-static void gfx_step(GfxExecStack& exec_stack) {
-    auto& cmd = exec_stack.currCmd();
-    auto cmd0 = cmd;
-    uint32_t opcode = cmd->words.w0 >> 24;
-
-    // if (markerOn)
-    // printf("OP: %016X\n", cmd0->force_structure_alignment);
+typedef bool (*GfxOpcodeHandlerFunc)(Gfx** gfx);
```

Comment:
it wasn't clear to me what this was for when i first saw it, adding some kind of comment or something explain a little might be nice